values.yaml

```diff
@@ -736,6 +736,23 @@ client:
   # This also changes the clients' advertised IP to the `hostIP` rather than `podIP`.
   exposeGossipPorts: false
 
+  ports:
+  # Customizes ports for the different client functionalities.
+    serflanTcp:
+      port: 8301
+    serflanUdp:
+      port: 8301
```

Comment:
You need to pass these ports to the client via the `ports` configuration. We're using `-hcl` flags for that (e.g. http, https and grpc ports).